beatmap_collections/static/css/index.css

```diff
@@ -34,6 +34,12 @@ a:hover {
     border-color: var(--bs-primary) !important;
 }
 
+h1 {
+    /* I want header 1 to be the only header that is bold */
+    font-family: "Comfortaa", cursive;
```

Comment:
If something happens, the web will be in cursive font like this.

<img width="563" alt="Screen Shot 2564-10-30 at 13 56 19" src="https://user-images.githubusercontent.com/37771900/139523820-ab675a85-42dd-4822-90d3-649c44f30a45.png">